GHzDACs/ghz_fpga_server.py

```diff
@@ -1855,8 +1963,46 @@ def dac_bist(self, c, chan, data):
         cmd, shift = dac.DAC.getCommand({'A': (2, 0), 'B': (3, 14)}, chan)
         dev = self.selectedDAC(c)
         ans = yield dev.runBIST(cmd, shift, data)
-        returnValue(ans)
-    
+        # This is coming back with 64-bit ints, the coercing of which needs to be fixed in pylabrad
+        # for now we manually cast to 32-bit (long)
+        # See pylabrad github issue #43
+        returnValue((ans[0],
+                    [long(x) for x in ans[1]], [long(x) for x in ans[2]], [long(x) for x in ans[3]],
```

Comment:
I'm surprised this works at all given that the return type is supposed to be `b(ww)(ww)(ww)`. We should explicitly create tuples:
```python
def coerce(xs):
    return tuple(long(x) for x in xs)
returnValue((ans[0], coerce(ans[1]), coerce(ans[2]), coerce(ans[3])))
```